internal/venice-common/src/main/java/com/linkedin/venice/blobtransfer/client/P2PFileTransferClientHandler.java

```diff
@@ -72,9 +72,11 @@ protected void channelRead0(ChannelHandlerContext ctx, HttpObject msg) throws Ex
 
       // Parse the file name
       this.fileName = getFileNameFromHeader(response);
+
       if (this.fileName == null) {
         throw new VeniceException("No file name specified in the response for " + payload.getFullResourceName());
       }
+      LOGGER.info("Starting blob transfer for file: {}", fileName);
```

Comment:
Can we make this `DEBUG`? It'd be a lot of logs for a single partition.

Reply:
But actually I am feeling it is pretty useful for debugging purpose...I am not sure if this matters or not... given that Da Vinci only usually subscribe to a few partitions?

Reply:
Or maybe once we make sure this is working in local QEI deploy, we can for now still make it as DEBUG.
Though I still think the log volume won't be too many, like ~20 per replica? Or maybe if it large hybrid store it can have a lot prior to compaction...

Reply:
> Or maybe once we make sure this is working in local QEI deploy, we can for now still make it as DEBUG

I'm cool with that. Right now, using INFO seems to be a lot easier for you to test it out. We can make it DEBUG afterwards

Reply:
should i put as as info or debug? as soon as I know I can push the new commit addressing the comments

Reply:
Yeah, for now logging means everything. We can take another pass when we are really pushing it toward production. The next step is the performance...which I have some concern but let's commit it fix and do the test again in VSC.